src/runtime/virtcontainers/qemu_test.go

```diff
@@ -636,3 +753,27 @@ func TestQemuSetConfig(t *testing.T) {
 
 	assert.Equal(q.config, config)
 }
+
+func TestQemuStartSandbox(t *testing.T) {
+	assert := assert.New(t)
+
+	sandbox, err := createQemuSandboxConfig()
+	assert.NoError(err)
+
+	network, err := NewNetwork()
+	assert.NoError(err)
+
+	q := &qemu{
+		config: HypervisorConfig{
+			VMStorePath:  sandbox.store.RunVMStoragePath(),
+			RunStorePath: sandbox.store.RunStoragePath(),
+		},
+		virtiofsDaemon: &virtiofsdMock{},
+	}
+
+	err = q.CreateVM(context.Background(), sandbox.id, network, &sandbox.config.HypervisorConfig)
+	assert.NoError(err)
+
+	err = q.StartVM(context.Background(), 10)
```

Comment:
Doesnt this actually start the qemu process? Make sure there are no processes left behind after the test is executed.

Reply:
No, this should actually error out with `failed to launch qemu: fork/exec /tmp/vc-tmp-623275751/hypervisor: permission denied`. That's why we have `assert.Error(err)`.

Reply:
My bad, overlooked the assert for error.